dali/imgcodec/decoders/decoder_test_helper.h

```diff
@@ -227,6 +226,12 @@ class DecoderTestBase : public ::testing::Test {
     }
   }
 
+  Tensor<CPUBackend> AsTensor(TensorView<StorageCPU, OutputType> img_view) {
+    Tensor<CPUBackend> img;
+    img.ShareData(img_view.data, 0, false, img_view.shape, type2id<OutputType>::value, -1);
+    return img;
+  }
+
```

Comment:
In the meantime the decoder test framework was [extended to support GPU decoders](https://github.com/NVIDIA/DALI/pull/4142) and as a result some functions are returning/getting `TensorView` instead of `Tensor`, while some still use `Tensor`s. I wrote this small util to be able to turn `TensorView` to `Tensor`. Is it OK? Do we already have something like that?

Reply:
The framework itself works just fine as long as we `Crop` the reference (which is a `Tensor`). In a single test (`GrayToRgb`) I crop the decoded image (which is a `TensorView`), and before passing it to `Crop` I need to dress it up as a `Tensor` and that's why I need this util. Probably more similar situations will appear in the future.

Reply:
@jantonguirao proposed a better solution in https://github.com/NVIDIA/DALI/pull/4154

Reply:
https://github.com/NVIDIA/DALI/pull/4154 should remove the need for this `AsTensor`